src/NgForage/session-storage/_isSessionStorageUsable.ts

```diff
@@ -2,5 +2,13 @@ import {checkIfSessionStorageThrows} from './checkIfSessionStorageThrows';
 
 /** @internal */
 export function _isSessionStorageUsable() {
```

Comment:
Why write 9 lines of code when you can write one instead to achieve the same thing.

```typescript
return typeof sessionStorage !== 'undefined' && (!checkIfSessionStorageThrows() || sessionStorage.length > 0);
```